src/shapes/ActiveSelection.spec.ts

```diff
@@ -55,4 +56,20 @@ describe('ActiveSelection', () => {
     selection.add(new FabricObject({ left: 50, top: 50, strokeWidth: 0 }));
     expect(selection.item(0).getCenterPoint()).toEqual({ x: 50, y: 50 });
   });
+
+  it('sets coords after attaching to canvas', () => {
+    const canvas = new Canvas(null, {
+      activeSelection: new ActiveSelection([
+        new FabricObject({
+          left: 100,
+          top: 100,
+          width: 100,
+          height: 100,
+        }),
+      ]),
+      viewportTransform: [2, 0, 0, 0.5, 400, 150],
+    });
+    expect(canvas.getActiveSelection().getCoords()).toMatchSnapshot();
```

Comment:
let's test by picking aCoords and lineCoords that is what are we testing today.
We are testing that those are correct, without the need of being recalculated.
getCoords return an array and is mostly for internal usage of calculations, if we snapshot aCoords and lineCoords we also see tl,tr,bl,br.

Reply:
> We are testing that those are correct, without the need of being recalculated.

They are not recalcuated.

I just have in mind #8767 so I avoid lineCoords already but whatever

Reply:
they are not recalculated no, but no owe knows it, is not written the test is not explcit.
if getCoords change or recalculation becomes default, this test doesn't break but also becomes uneffective.

Reply:
I do not follow
What should be changed?

Reply:
oh i thought you changed because i have seen outdated tag on the discussion
i ll change it never mind